src/gg.regression.unity.bots/Runtime/Scripts/CodeCoverage/RGCodeCoverage.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json;
+using UnityEngine;
+
+namespace RegressionGames.CodeCoverage
+{
+    /// <summary>
+    /// Class responsible for tracking code coverage.
+    /// </summary>
+    public static class RGCodeCoverage
+    {
+        public static readonly string CodeCovMetadataJsonPath = "Assets/Resources/RGCodeCoverageMetadata.txt";
```

Comment:
I feel like this file should be saved with the recording data as it will vary from run to run when code changes have been made in the project.

Reply:
I've added code to save the metadata to the session recording folder. However, the specific line pointed out here is where the metadata gets initially generated during the instrumentation (this only happens within the Unity Editor when our hooks activate on Unity's assembly build pipeline), so this part will remain the same. 

I intentionally have it go to the `Resources` folder so that in production runtimes / standalone builds, the metadata can be retrieved via the Resources API.